modules/data.atmosphere/R/download.AmerifluxLBL.R

```diff
@@ -19,10 +19,13 @@
 ##' 
 ##' @author Ankur Desai, based on download.Ameriflux.R by Josh Mantooth, Rob Kooper
 download.AmerifluxLBL <- function(sitename, outfolder, start_date, end_date, 
-                                  overwrite = FALSE, verbose = FALSE, username = "pecan", ...) {
+                                  overwrite = FALSE, verbose = FALSE, username = "pecan", method=NULL, ...) {
   # get start/end year code works on whole years only
   
-  
+  # set hostname to select appropriate download
+  #if (is.null(host)) host <- system("hostname", intern=T)
+  if (system("hostname", intern=T)=="modex") method <- "ncftpget"  
```

Comment:
OK, I agree....this was an attempt to get things working and get feedback on options to generalize